src/go/rpk/pkg/cli/cmd/common/common.go

```diff
@@ -510,6 +558,39 @@ func AddTLSFlags(
 	return command
 }
 
+func AddAdminAPITLSFlags(
+	command *cobra.Command,
+	enableTLS *bool,
+	certFile, keyFile, truststoreFile *string,
+) *cobra.Command {
+	command.PersistentFlags().BoolVar(
+		enableTLS,
+		adminAPIEnableTLSFlag,
+		false,
+		"Enable TLS for the Admin API.",
+	)
+	command.PersistentFlags().StringVar(
+		certFile,
+		adminAPICertFileFlag,
+		"",
+		"The certificate to be used for TLS authentication with the Admin API.",
+	)
+	command.PersistentFlags().StringVar(
+		keyFile,
+		adminAPIKeyFileFlag,
+		"",
+		"The certificate key to be used for TLS authentication with the Admin API.",
+	)
+	command.PersistentFlags().StringVar(
+		truststoreFile,
+		adminAPITruststoreFileFlag,
+		"",
+		"The truststore to be used for TLS communication with the Admin API.",
```

Comment:
I look forward to the day that `truststore` is only ever used in context with Java specifically :|

Reply:
We can add aliases for these in a later commit.